docs/tutorials/practice-your-learning.md

```diff
@@ -1,4 +1,4 @@
-# Put into Practice
+# Putting It All Together
```

Comment:
please revert this change and make this file stays in the original position

Reply:
Done